Oro 3.1 upgrade

krasilich opened the pull request.

krasilich committed:
Change project structure

krasilich committed:
Change project structure

krasilich committed:
Change project structure

krasilich committed:
- Fix upgrade issues

sashaskr committed:
DIAM-1915 Removed unused Web UI installer

- Removed unused classes from DistributionBundle
- Removed *.twig files

Signed-off-by: sashaskr <shurikskr@hotmail.com>

sashaskr committed:
DIAM-1916 Fix DeskBundle services

- Changed oro_datagrid.event_listener.base_orm_relation.class to oro_datagrid.event_listener.row_selection.class as base_orm_relation is deprecated since version since 1.4.1 in favor of RowSelectionListener and "bind_parameters" option in source

Signed-off-by: sashaskr <shurikskr@hotmail.com>

sashaskr committed:
DIAM-1917 Replace DefinitionDecorator

- Replaced DefinitionDecorator which is deprecated since version 3.3 and will be removed in 4.0 with ChildDefinition class

Signed-off-by: sashaskr <shurikskr@hotmail.com>

sashaskr committed:
DIAM-1919 Replaced functionality on InstallCommand class

- New Command executable
- Replaced checkRequirementsStep with runCommand
TODO:
- Verbosity logic on renderTable.
- refactor code. Delete unused methods

Signed-off-by: sashaskr <shurikskr@hotmail.com>

sashaskr committed:
DIAM-1919 Removed unused code

krasilich committed:
- Update travis config

krasilich committed:
- Fix upgrade issues

krasilich committed:
Update .travis.yml

krasilich committed:
Merge pull request #83 from sashaskr/DIAM-1915

DIAM-1915 Removed unused Web UI installer

krasilich committed:
Merge pull request #85 from sashaskr/DIAM-1917

DIAM-1917 Replace DefinitionDecorator

krasilich committed:
Merge pull request #84 from sashaskr/DIAM-1916

DIAM-1916 Fix DeskBundle services

sashaskr committed:
DIAM-1919 Cherry-pick travis from master

sashaskr committed:
DIAM-1919 Added -vv parameter in runCommand

krasilich committed:
Merge pull request #86 from sashaskr/DIAM-1919

Diam 1919

sashaskr committed:
DIAM-1922 Updated syntax for assets.yml

- Updated syntax on assets.yml files
- Commented assets files where extension is *.less

sashaskr committed:
Merge branch 'oro-3.1-upgrade' into DIAM-1922

sashaskr committed:
DIAM-1923 Fixed get() method

krasilich committed:
Merge pull request #87 from sashaskr/DIAM-1922

DIAM-1922 Upgraded assets

sashaskr committed:
WIP

- changed the inhiretance for loadData

sashaskr committed:
Merge branch 'oro-3.1-upgrade' into DIAM-1293-TBC

sashaskr committed:
Renamed assets with less

sashaskr committed:
Fixed OptioinResolver, and datagrid

sashaskr committed:
Changed types

TODO: routes.json -> _routes.json

sashaskr committed:
Added support to assert

sashaskr committed:
Fixed ticket create, ticket update WIP

sashaskr committed:
Unchained ticketBuilder

sashaskr committed:
WIP

sashaskr committed:
Fixed the ticket controller

Fixed:
- View
- Create
- Update
TODO: Delete

sashaskr committed:
Commented clearSearchIndex() method

Commented, unless the method will be fixed

sashaskr committed:
Fixed assign functionality

sashaskr committed:
Fixed ticket status change

sashaskr committed:
Fixed some errors on Watcher

sashaskr committed:
Fixed types and controller for Attachments

sashaskr committed:
fixes on branch

Fixed:
- Branch controller
- Types
- DoctrineHelper argument. Related to DIAM-1916

sashaskr committed:
Fixed branches deletion and updating

krasilich committed:
Merge pull request #88 from sashaskr/DIAM-1923

DIAM-1923 Fixed get() method

sashaskr committed:
DIAM-1925 Fixed types and OptionResolver

- Fixed class declaration from string to ::class as required in Symfony 3.*
- Changed OptionResolverInterface to OptionResolver

sashaskr committed:
DIAM-1925 Fixed types

- Added getBlockPrefix() instead of getName()
- Added support of Assert functionality

sashaskr committed:
DIAM-1925 Fixed transformers

- Fixed transformers. In the Symfony 3.* keys and values for getOptions were swapped
- Unchained ticketBuilder

sashaskr committed:
DIAM-1925 Fixed controllers and types

- Fixed the last part of the types
- Fixed controller

krasilich committed:
Merge pull request #89 from sashaskr/DIAM-1925-fixing-types-and-option-resolver

DIAM-1925 fixing types and option resolver

krasilich committed:
Merge pull request #90 from sashaskr/DIAM-1926-fix-update-delete-branch

DIAM-1926 Fixed update and delete functionality

sylvainraye commented:
Hello there, what is the status of the upgrade to 3.1 ? Looks like you are still active about this project, right ? I'm asking because we are also working in parallel on the upgrade.

krasilich commented:
Hello @sylvainraye
I appreciate your contributions, unfortunately this branch is not up to date with our current progress. Most of things you submitted are resolved as of now, so I'll close your PR's. Could you please provide the preferred method of communication (emails. messengers...)  so we can collaborate more effectively if you really want to help.
Thanks

sylvainraye commented:
Hi @krasilich
I tried to connect with you via linkedin however we can share on the slack of oro community if you are in, I would not like to share my email here publicly

sylvainraye commented:
Hi @krasilich please allow me to bump you. We will pleased to go forward on this topic. Thank you for your feedback

Commit:
DIAM-1927 Fixing the automation bundle
DIAM-1929 Fixing the Form Types to fit the Symfony 3 requirements
DIAM-1930 Fix the key=>value flip for new functionality of ‘choice’ type in branch entity
DIAM-1931 Fix datagrids
DIAM-1932 Fix Event Listeners
DIAM-1936 Fixing the controllers

Commit:
DIAM-1928 Fixing the UserBundle(minor changes) Embedded Form (minor changes), Emails Processing bundle
DIAM-1933 The next part of fixing Embedded form
DIAM-1934 Fix the comments
DIAM-1935 Change the cascade_validation to ‘constraints’ => new Valid()
DIAM-1937 Fixing navigation
DIAM-1938 Fix system_configuration.yml
DIAM-1939 Fix the embedded forms

krasilich committed:
Merge pull request #100 from eltrino/DIAM-1914

Diam 1914

Commit:
DIAM-1914 Fix config

Commit:
DIAM-1914
 - Add calendar bundle

Commit:
DIAM-1967: bug with name file routes.json
- Added configuration for routing filename prefix to config.yml

Commit:
DIAM-1943: Sidebar widget can't be added
- Added routes to whitelist

krasilich committed:
Merge pull request #103 from eltrino/DIAM-1967

DIAM-1967: bug with name file routes.json

krasilich committed:
Merge pull request #104 from eltrino/DIAM-1943

DIAM-1943: Sidebar widget can't be added

Commit:
DIAM-1947: Can't delete uploaded image at Branches
- Providing an empty logo file to branch update for successfully delete branch logo

krasilich committed:
Merge pull request #105 from eltrino/DIAM-1947

DIAM-1947: Can't delete uploaded image at Branches

Commit:
DIAM-1949: Errors with tickets' images interactions
- Edited csrf token name
- Changed upload images directory for attachments
- FIxed error with generated preview link with non existent route
- Added upload directories to gitignore

Commit:
[DIAN-1950] - remove task comments numbering

Commit:
[DIAN-1954,1948] - fix diam-user-type label

Commit:
DIAM-1966: DB error when clicking on the arrow to uncover sidebar widget's menu
- Added route to whitelist

Commit:
DIAM-1960: Database errors in Customers section
- Edited configuration in the datagrid.yml
- Added new helper service to return query builder for branch filter

Commit:
[DIAN-1945] - separate Manage Dashboard in Dashboards

krasilich committed:
Merge pull request #107 from eltrino/DIAM-1950

[DIAN-1950] - remove task comments numbering

krasilich committed:
Merge pull request #108 from eltrino/DIAM-1966

DIAM-1966: error when clicking on the arrow to uncover sidebar widget's menu

krasilich committed:
Merge pull request #109 from eltrino/DIAM-1960

DIAM-1960: Database errors in Customers section

Commit:
[DIAN-1957] - in navigation panel changed "reports" icon

Commit:
DIAM-1960: Database errors in Customers section
- Added new route to whitelist

Commit:
[DIAN-1957] - in navigation panel changed .active color

Commit:
[DIAN-1946] - vertical centered modal dialog

Commit:
DIAM-1970: Exception during customer's update
- Added try/catch block to catch exception when "isDeleted" value was not changed

krasilich committed:
Merge pull request #111 from eltrino/DIAM-1957

[DIAN-1957, 1946] style fixes

krasilich committed:
Merge pull request #112 from eltrino/DIAM-1970

DIAM-1970: Exception during customer's update

Commit:
DIAM-1971: Display settings couldn’t be saved
- Removed activity list from form clean-list

Commit:
DIAM-1975: Can't disable user
- Added route to the whitelist

Commit:
DIAM-1977: User edit is not saved
- Added route to the whitelist

Commit:
DIAM-1983: DB error in Embedded forms
- Added type attribute into diamante embedded form

Commit:
DIAM-1984: Error in Data audit section
- Injected resolver dependency in sorter extension
- Removed visitDatasource method from sorter extension and use parent method instead

Commit:
DIAM-1961: Non-existing currencies
- Added choice type extension for removing non-existing currency

Commit:
DIAM-1985: DB error durind interactions at side widget menu
- Added routes to the whitelist

Commit:
DIAM-1974: Unclear 1/0 options for choosing
- Changed project yaml configuration to show correct select options for channels and notifications config sections

krasilich committed:
Merge pull request #120 from eltrino/DIAM-1974

DIAM-1974: Unclear 1/0 options for choosing

krasilich committed:
Merge pull request #119 from eltrino/DIAM-1985

DIAM-1985: DB error durind interactions at side widget menu

krasilich committed:
Merge pull request #118 from eltrino/DIAM-1961

DIAM-1961: Non-existing currencies

krasilich committed:
Merge pull request #117 from eltrino/DIAM-1984

DIAM-1984: Error in Data audit section

krasilich committed:
Merge pull request #116 from eltrino/DIAM-1983

DIAM-1983: DB error in Embedded forms

krasilich committed:
Merge pull request #115 from eltrino/DIAM-1977

DIAM-1977: User edit is not saved

krasilich committed:
Merge pull request #114 from eltrino/DIAM-1975

DIAM-1975: Can't disable user

krasilich committed:
Merge pull request #113 from eltrino/DIAM-1971

DIAM-1971: Display settings couldn’t be saved

krasilich committed:
Merge branch 'oro-3.1-upgrade' into DIAM-1960

krasilich committed:
Merge pull request #110 from eltrino/DIAM-1960

DIAM-1960: Database errors in Customers section

Commit:
Merge branch 'oro-3.1-upgrade' of github.com:eltrino/diamantedesk-application into DIAM-1949

Commit:
DIAM-1949: Errors with tickets' images interactions
- Added new data_root dir to liip_imagine configuration
- Changed upload path to var/attachments

Commit:
DIAM-1949: Errors with tickets' images interactions
- Removed unusable class

krasilich committed:
Merge pull request #106 from eltrino/DIAM-1949

DIAM-1949: Errors with tickets' images interactions

Commit:
[DIAN-1965] - added Diamante logo on 404 page, fixed "go back" button style

Commit:
[DIAN-1965] - fixed 404 page logo img

Commit:
Merge branch 'oro-3.1-upgrade' into DIAM-1965

Commit:
DIAM-1951: Can't see attached photo in comments
- Fixed uploading and removing comment attachments

krasilich committed:
Merge pull request #122 from eltrino/DIAM-1951

DIAM-1951: Can't see attached photo in comments

krasilich committed:
Merge pull request #121 from eltrino/DIAM-1965

[DIAM-1965] added Diamante logo on 404 page, fixed "go back" button style

Commit:
DIAM-1969: Need to install the DiamanteDesk
- Refactor diamante install command

Commit:
[DIAN-1958] - fix rules creating pages (add css and assets to automation bundle)

Commit:
Merge branch 'oro-3.1-upgrade' into DIAM-1958

Commit:
Merge branch 'oro-3.1-upgrade' of github.com:eltrino/diamantedesk-application into DIAM-1969

Commit:
DIAM-1969:
- Added param to method

Commit:
DIAM-1990: API errors
- Fixed undefined 'request' service, getting from request stack
- Removed getting diamante user from logged user
- Added required param to the method 'removeAttachmentFromComment'

Commit:
DIAM-1989: Error when run cron console command
- Removed running daemon because service 'oro_cron.job_daemon' and class 'Oro\Bundle\CronBundle\Job\Daemon' was removed from OROPlatform

krasilich committed:
Merge pull request #123 from eltrino/DIAM-1958

[DIAN-1958] fix rules creating pages styles

krasilich committed:
Merge pull request #124 from eltrino/DIAM-1969

Diam 1969

krasilich committed:
Merge pull request #126 from eltrino/DIAM-1989

DIAM-1989: Error when run cron console command

krasilich committed:
Merge pull request #125 from eltrino/DIAM-1990

DIAM-1990: API errors

Commit:
DIAM-1995: Multiple errors during configuring sidebar widgets - Recent emails
- Added route to the whitelist

Commit:
[DIAN-1996] - fixed markup bug during branch creating

Commit:
[DIAM-1996] - fixed markup bug during branch creating

Commit:
Merge branch 'DIAM-1996' of https://github.com/eltrino/diamantedesk-application into DIAM-1996

krasilich committed:
Merge pull request #128 from eltrino/DIAM-1996

[DIAM-1996]

krasilich committed:
Merge pull request #127 from eltrino/DIAM-1995

DIAM-1995: Multiple errors during configuring sidebar widgets - Recen…

Commit:
[DIAN-2006] - fixed Markup bug after adding watchers to a ticket

Commit:
Merge branch 'oro-3.1-upgrade' into DIAM-1996

Commit:
[DIAM-2005] - added indent at comment section

Commit:
[DIAN-1998] - Bug during mass branches deletion

Commit:
[DIAN-1997] - fixed pop-up window bug during branch deletion

Commit:
[DIAN-1994] - Position of pop-up window

krasilich committed:
Merge pull request #130 from eltrino/DIAM-1998

[DIAM-1998, DIAM-1997, DIAM-1994]

krasilich committed:
Merge branch 'oro-3.1-upgrade' into DIAM-1996

krasilich committed:
Merge pull request #129 from eltrino/DIAM-1996

[DIAМ-2006] - fixed Markup bug after adding watchers to a ticket

Commit:
DIAM-1914
 - fix merge

Commit:
DIAM-2002
DIAM-2003
DIAM-2004
 - fix form file upload

krasilich committed:
Merge pull request #131 from eltrino/DIAM-2002

DIAM-2002

Commit:
DIAM-2004
 - fix duplicate attachments upload

krasilich committed:
Merge pull request #132 from eltrino/DIAM-2002

DIAM-2004

Commit:
DIAM-1984
 - fix error message on grid

krasilich committed:
Merge pull request #133 from eltrino/DIAM-1984

DIAM-1984

Commit:
DIAM-1914
 - fix merge

krasilich committed:
Merge pull request #134 from eltrino/DIAM-1998

DIAM-1914

Commit:
DIAM-1953
 - fix settings default branch form type

krasilich committed:
Merge pull request #135 from eltrino/DIAM-1953

DIAM-1953

Commit:
DIAM-1920
DIAM-1921
 - fix issue with extended fields reference

krasilich committed:
Merge pull request #136 from eltrino/DIAM-1920

DIAM-1920

krasilich committed:
DIAM-1920
DIAM-1921
 - fix issue with extended fields reference
